src/pages/Vote/CreateProposalModal/index.tsx

```diff
@@ -49,6 +50,17 @@ export const CreateProposal: React.FC<CreateProposalProps> = ({
         title: t('vote.pages.proposalInformation'),
         Component: () => (
           <>
+            <FormikSelectField
+              name="proposalType"
+              title={t('vote.createProposalForm.proposalType')}
+              ariaLabel={t('vote.createProposalForm.proposalType')}
+              css={styles.formBottomMargin}
```

Comment:
You'll need to add a label saying "Proposal type" here.
<img width="358" alt="Screen Shot 2022-12-23 at 12 23 08" src="https://user-images.githubusercontent.com/44675210/209327774-68303c40-dd3a-4772-a2c1-6bc34fe5c783.png">

Reply:
ahh yeah I missed the select only has a mobile header